docs/administration-guide/src/main/asciidoc/overview.adoc

```diff
@@ -800,310 +799,6 @@ add-on components. For more information, see
 xref:general-administration.adoc#using-rest-interfaces-to-administer-glassfish-server[Using REST Interfaces to
 Administer {productName}].
 
-[[osgi-module-management-subsystem]]
-
-==== OSGi Module Management Subsystem
-
-The OSGi module management subsystem that is provided with {productName} is the http://felix.apache.org/[Apache Felix OSGi framework] . To
-administer this framework, use the either of the following tools:
-
-* http://felix.apache.org/documentation/subprojects/apache-felix-remote-shell.html[Apache
-Felix Gogo] remote shell. This shell is provided with {productName}.
-The shell uses the Felix Gogo shell service to interact with the OSGi
-module management subsystem.
-* GlassFish OSGi Administration Console. This console is distributed as
-an add-on component for {productName} or as a set of files from the
-Maven GlassFish repository. In both distributions, the GlassFish OSGi
-Web Console is provided as an extension to the Administration Console
```

Comment:
Remove only the following sentences about Web Console:
* remove sentence starting with "In both distributions, the GlassFish OSGi"
* remove sentence starting with "The GlassFish OSGi Administration
Console"

Keep the rest of the section.